### PR TITLE
Fix test_eig_arange

### PR DESCRIPTION
**Pull request by LukichevaPolina**

### CODE REVIEW

**Comment by Alexander-Makaryev** [comment]
@LukichevaPolina Could you please keep the old version of vvsort as commented code and provide some description why it is replaced in comment also?

File: tests/test_linalg.py [hunk 1]
```diff
@@ -6,18 +6,41 @@
...
                 imax = j
 
-        temp = val[i]
-        val[i] = val[imax]
-        val[imax] = temp
+        unravel_i = numpy.unravel_index(i, val.shape)
+        unravel_imax = numpy.unravel_index(imax, val.shape)
 
+        temp = xp.empty(tuple())
+        temp[()] = val[unravel_i] # make a copy
```
Comment (densmirn) [inline tests/test_linalg.py:37]:
```suggestion
        temp[()] = val[unravel_i]  # make a copy
```

File: tests/test_linalg.py [hunk 2]
```diff
@@ -6,18 +6,41 @@
...
+        val[unravel_i] = val[unravel_imax]
+        val[unravel_imax] = temp
+        
         for k in range(size):
-            temp = vec[k, i]
+            temp = xp.empty(tuple())
+            temp[()] = vec[k, i] # make a copy
```
Comment (densmirn) [inline tests/test_linalg.py:43]:
```suggestion
            temp[()] = vec[k, i]  # make a copy
```

File: tests/test_linalg.py [hunk 3]
```diff
@@ -5,19 +5,26 @@
 import numpy
 
 
-def vvsort(val, vec, size, xp):
+def vvsort(val, vec, size, xp):
```
Comment (densmirn) [inline tests/test_linalg.py:8]:
```suggestion
def vvsort(val, vec, size, xp):
```

**Review by densmirn — APPROVED** [review]

**Review by Alexander-Makaryev — APPROVED** [review]
LGTM

**Pull request merged**